test/trajectories/transition.jl

```diff
@@ -11,17 +11,17 @@
 
     fp1 = [0.816, 0.272]
     fp2 = [-0.816, -0.272]
-    trajectory, time, succes = CT.transition(sys, fp1, fp2)
+    tr, time, succes = CT.transition(sys, fp1, fp2)
     @test succes
     @test time[end] < 1e3
-    @test norm(trajectory[:, 1] - fp1) < 0.1
-    @test norm(trajectory[:, end] - fp2) < 0.1
+    @test norm(tr[1, :] - fp1) < 0.1
+    @test norm(tr[end, :] - fp2) < 0.1
 
     ensemble = transitions(sys, fp1, fp2, 10)
     @test ensemble.success_rate ≈ 1.0
     @test ensemble.t_trans ≈ 4.493941793363376 atol = 1e-2
```

Comment:
Doesn't this give very different numbers each time? `atol=1e-2` seems very small

Reply:
Normally with the Seed it should give the same every time

Reply:
Ah, ok. I thought seed didn't work on github actions? This now resolved?

Reply:
It seems to work now on CI 😊